client/galaxy/scripts/mvc/dataset/dataset-edit-attributes.js

```diff
@@ -0,0 +1,250 @@
+
+define( [ 'utils/utils', 'mvc/ui/ui-tabs', 'mvc/ui/ui-misc', 'mvc/form/form-view' ], function( Utils, Tabs, Ui, Form ) {
+
+    /** Dataset edit attributes view */
+    var View = Backbone.View.extend({
+
+        initialize: function( dataset_id ) {
+            this.setElement( '<div/>' );
+            this.render( dataset_id );
```

Comment:
The render function should not require a parameter. Instead we could populate a model e.g. this.model = new Backbone.Model( options ) in the initialize method and then use the model in the render function.